tests/terratest/azure_acr_test.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformAcr(t *testing.T) {
+	t.Parallel()
+
+	terraformOptions := &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: "../../example",
+		//VarFiles:     []string{"terratest.tfvars"},
+		//Upgrade:      true,
+	}
+
+	// Defer the destroy to cleanup all created resources
+	defer terraform.Destroy(t, terraformOptions)
+
+	// This will init and apply the resources and fail the test if there are any errors
+	terraform.InitAndApply(t, terraformOptions)
+
+	// Run `terraform output` to get the values of output variables
+	RG := terraform.Output(t, terraformOptions, "acr")
```

Comment:
I dont see acr in the TF outputs.
I have to be ACR_NAME.
Also update the variable RG to something like ACRName and use the same in assert.